Added Kriten.

beertwanger opened the pull request:
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [ ] Have you followed the guidelines in our Contributing document in setting up the application?
* [ ] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [ ] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [ ] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?
* [ ] Have you included the specific version of the application to be installed?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [ ] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [ ] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot).
* [ ] Updated the version number of the app if that has changed.

saiyam1814 commented on kriten/manifeat.yaml:
```diff
@@ -0,0 +1,6 @@
+name: Kriten
```
lower case please kriten

beertwanger commented:
I’ve renamed the file manifeat.yaml -> manifest.yaml and changed to lower case “kriten”

Regards,
Steve

From: Saiyam Pathak ***@***.***>
Date: Thursday, 29 February 2024 at 08:34
To: civo/kubernetes-marketplace ***@***.***>
Cc: Steve Corp ***@***.***>, Author ***@***.***>
Subject: Re: [civo/kubernetes-marketplace] Added Kriten. (PR #689)

@saiyam1814 commented on this pull request.

________________________________

In kriten/manifeat.yaml<https://github.com/civo/kubernetes-marketplace/pull/689#discussion_r1507199998>:

> @@ -0,0 +1,6 @@

+name: Kriten

lower case please kriten

—
Reply to this email directly, view it on GitHub<https://github.com/civo/kubernetes-marketplace/pull/689#pullrequestreview-1908266578>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AB5XWTBL5AULE2OAGYNCWPTYV3T2HAVCNFSM6AAAAABDVMCLUWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTSMBYGI3DMNJXHA>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
